src/commands/readability.ts

```diff
@@ -0,0 +1,143 @@
+import { CommandModule } from "yargs";
+import { promises as fs } from "fs";
+import restructured, { AnyNode } from "../restructured";
+import MagicString from "magic-string";
+import { findAll, visit } from "../tree";
+import toml from "toml";
+import * as path from "path";
+
+type SnootyConfig = {
+  constants: Record<string, string>;
+};
+
+const loadSnootyConfig = async (
+  snootyTomlPath?: string
+): Promise<SnootyConfig> => {
+  const defaults: SnootyConfig = {
+    constants: {},
+  };
+  if (snootyTomlPath === undefined) {
+    return { ...defaults };
+  }
+  const text = await fs.readFile(snootyTomlPath, "utf8");
+  const data = toml.parse(text);
+  return { ...defaults, ...data } as SnootyConfig;
+};
+
+// https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+// Note: = reused, but uses top title for first section depth
+const titleAdornmentCharacters = ["=", "-", "~", "^", "`", "_", "="];
+
+const getText = (args: {
+  inputPath: string;
+  document: MagicString;
+  rst: AnyNode;
+  snootyConfig: SnootyConfig;
+}) => {
+  const { inputPath, rst, snootyConfig } = args;
+  const desiredText: string[] = [];
+  visit(rst, (node) => {
+    switch (node.type) {
+      case "title": {
+        const textNodes = findAll(node, (n) => n.type === "text");
+        let text = textNodes.map((textNode) => textNode.value).join("");
+        // If the last character of the title is not a period, add one.
+        // Missing periods will negatively impact readability.
+        const lastTitleChar = text.charAt(text.length - 1);
+        if (lastTitleChar != ".") {
+          text = text + ".";
+        }
+        desiredText.push(text + "\n");
+        break;
+      }
+      // TODO: Look into whether newlines in the paragraphs are inflating 
+      // readability scores to be artificially high.
+      // If so, remove newlines in paragraphs and directives nodes.
+      case "paragraph": {
+        const textNodes = findAll(node, (n) => n.type === "text");
+        const text = textNodes.map((textNode) => textNode.value).join("");
+        // Remove the <some-anchor-tag> markup that the rST parsing leaves 
+        // in the plain text, as this skews readability.
+        const unwantedText = new RegExp("<.*?>");
+        desiredText.push(text.replace(unwantedText, ""));
+        break;
+      }
+      // TODO: Clean up directive output. Check for tables, code blocks,
+      // and other unwanted elements that will skew readability scores.
+      case "directive": {
+        const textNodes = findAll(node, (n) => n.type === "text");
+        const text = textNodes.map((textNode) => textNode.value).join("");
+        // Remove the <some-anchor-tag> markup that the rST parsing leaves 
+        // in the plain text, as this skews readability.
+        const unwantedText = new RegExp("<.*?>");
```

Comment:
You can use regex literal:
```suggestion
        const unwantedText = /<.*?>/;
```